administrator/language/en-GB/com_fields.ini

```
@@ -59,6 +59,8 @@ COM_FIELDS_FIELD_REQUIRED_LABEL="Required"
 COM_FIELDS_FIELD_SAVE_SUCCESS="Field saved"
 COM_FIELDS_FIELD_SHOWLABEL_DESC="Show or Hide the label when the field displays."
 COM_FIELDS_FIELD_SHOWLABEL_LABEL="Label"
+COM_FIELDS_FIELD_SHOWON_LABEL="Showon Attribute"
+COM_FIELDS_FIELD_SHOWON_DESC="Conditionally show or hide the field depending on other field values"
```

```suggestion
COM_FIELDS_FIELD_SHOWON_DESC="Conditionally show or hide the field depending on the value of other fields."
```

sentence ends with a full stop

---

Thank you @brianteeman